### DISCUSSION TITLE
DPI

### DISCUSSION THREAD

**Discussion by mfaqiri** [X3n0m0rph59/eruption#35] [Q&A]
Is this project only focused on LED functionality or will dpi be looked at later?

**Comment by X3n0m0rph59** [comment]
Hi! Currently the project is primarily focused on providing LED effects support and macros.
It is certainly possible to add support for device specific parameters like angle snapping, distance control or DPI levels in the future! In the meantime, if you want to change the DPI settings you could look into this project: https://github.com/libratbag

**Reply by mfaqiri** [reply]
Thank you
